tests/copyoffload/conftest.py

```
@@ -0,0 +1,356 @@
+        string_data=secret_data,
+    )
+
+    LOGGER.info(f"✓ Copy-offload storage secret created: {storage_secret.name}")
+    return storage_secret
+
+
+@pytest.fixture(scope="session")
+def copyoffload_ssh_key(
+    source_provider: VMWareProvider,
+    source_provider_data: dict[str, Any],
+    copyoffload_config: None,
```

Ruff (0.15.5)

[warning] Unused function argument: `copyoffload_config`

(ARG001)